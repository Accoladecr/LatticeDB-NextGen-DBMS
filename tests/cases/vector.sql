CREATE TABLE vecs (
                      id TEXT PRIMARY KEY,
                      v VECTOR<3>
);
INSERT INTO vecs (id, v) VALUES
                             ('a', [0.1,0.0,0.0]),
                             ('b', [0.3,0.0,0.0]);
SELECT id FROM vecs WHERE DISTANCE(v, [0.0,0.0,0.0]) < 0.2;
EXIT;
