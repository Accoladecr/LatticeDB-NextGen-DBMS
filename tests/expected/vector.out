CREATE TABLE
INSERT 2 row(s)
id
'a'
Bye.
